packages/marlinkinfit/package.py

```diff
@@ -44,4 +44,4 @@ def cmake_args(self):
     def setup_run_environment(self, env):
         env.prepend_path("MARLIN_DLL", self.prefix.lib + "/libMarlinKinfit.so")
         # Make it usable from ROOT
-        env.prepend_path("LD_LIBRARY_PATH", self.prefix.lib)
+        env.prepend_path("ROOT_LIBRARY_PATH", self.prefix.lib)
```

Comment:
Is `ROOT_LIBRARY_PATH` a new thing that root uses for looking for dictionaries / libraries to load? Or is this a place where things can be put, but will not be loaded automatically?

Reply:
Not new but hard to spot. ROOT looks there so loading `libMarlinKinFit.so` will work as if it was in `LD_LIBRARY_PATH`. I'll move some paths over time from `LD_LIBRARY_PATH` to `ROOT_LIBRARY_PATH`.